gke/standard/zonal/arm/main.tf

```diff
@@ -0,0 +1,55 @@
+/**
+* Copyright 2024 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START gke_standard_zonal_arm]
+resource "google_service_account" "default" {
+  account_id   = "service-account-id"
+  display_name = "Service Account"
+}
+
+# [START gke_standard_zonal_arm_cluster]
+resource "google_container_cluster" "default" {
+  name               = "gke-standard-zonal-arm-cluster"
+  location           = "us-central1-a"
+  node_locations     = ["us-central1-b", "us-central1-b"]
```

Comment:
two us-central1-b ?

Reply:
:) Seems to work, but yeah, the intention was to use `a` and `b` as they support ARM nodes.